src/index.ts

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import request from "request";
+import * as request from "request";
```

Comment:
This broke our client. `request` here now is an object containing `get`, `post`, etc, instead of a function.
Repro:
```js
const { getRequest } = require('matrix-js-sdk'); // notice we're importing from index.js, as expected
typeof getRequest(); // gives `object`, instead of `function`, as before
```

Reply:
It seems the transpiler helper `_interopRequireWildcard` converts the function to object